adoc/deployment-bootstrap.adoc

```diff
@@ -70,6 +70,65 @@ The IP/FQDN must be reachable by every node of the cluster and therefore 127.0.0
 
 At this point you can perform some configurations before bootstrapping the cluster.
 
+===== Enable cloud provider integration
+
+Enable cloud provider integration to take advantage of the underlying cloud platforms 
+to automatically manage resources like Load Balancer, Nodes (Instances), Network Routes 
+and Storage services. +
+If you want to enable cloud provider integration with different cloud platforms, 
+you have to initialize the cluster with flag `--cloud-provider <CLOUD PROVIDER>`. +
+Currently it is available only for `openstack` but more possibility will be added soon.
+
+----
+skuba cluster init --control-plane <LB IP/FQDN> --cloud-provider openstack my-cluster
+----
+
+It will create the directory `my-cluster/cloud/openstack` with `README.md` and `openstack.conf.template` in it.
+You have to copy `openstack.conf.template` or create a `openstack.conf` file inside `my-cluster/cloud/openstack`, 
```

Comment:
nit: as you state `or create` you might want to specify what file permissions are expected? I might also refer to it as `<my-cluster>` as this might be more inline with how we document variable values.

Reply:
done